calendar-service/src/main/java/org/exoplatform/calendar/service/CalendarService.java

```diff
@@ -835,6 +835,16 @@ default public boolean hasCalendarSetting(String username) throws Exception {
    */
   public void savePublicEvent(String calendarId, CalendarEvent event, boolean isNew) throws Exception;
 
+  /**
+   * Saves event or task to a public calendar
+   * @param username The user who's saving the event
+   * @param calendarId Id of the public calendar to which the event will be saved
+   * @param event <code>CalendarEvent</code> object to be saved.
+   * @param isNew If <code>true</code>, a new event will be saved. If <code>false</code>, an existing event will be updated.
+   * @throws Exception
+   */
+  public void savePublicEvent(String username, String calendarId, CalendarEvent event, boolean isNew) throws Exception;

```

Comment:
Add default to avoid being forced to update all implementations.
```suggestion
  public default void savePublicEvent(String username, String calendarId, CalendarEvent event, boolean isNew) throws Exception {}

```

Reply:
Done !!!